graylog2-web-interface/src/components/alarmcallbacks/AlarmCallbackHistory.jsx

```diff
@@ -16,7 +16,6 @@ class AlarmCallbackHistory extends React.Component {
     const history = this.props.alarmCallbackHistory;
     const configuration = history.alarmcallbackconfiguration;
     const type = this.props.types[configuration.type];
-
```

Comment:
IMO adding additional blank lines to structure assignment blocks should be allowed.